### PR TITLE
Removed snowflake subtype of flash/handheld are placed with regular flash

### PR DESCRIPTION
**Pull request by ghost**
Removes the needless flash/handheld class and makes all code just use flash.
This fixes stuff like flashes not fitting in security belts or cyborg heads because it's the wrong type of flash.

### CODE REVIEW

**Comment by ghost** [comment]
Oh yeah, this certainly doesn't fuck putting cyborg flashes in cyborg heads.

**Comment by JohnGinnane** [comment]
Let me get this straight: we had a subtype of flash called handheld that was no different from it's parent?

**Comment by ghost** [comment]
Yes sir.
https://github.com/HippieStationCode/HippieStation13/blob/adaa12bf59f042195ea224a9a9a54e951a10ce79/code/modules/assembly/flash.dm#L239

**Comment by JohnGinnane** [comment]
For what purpose?

**Comment by ghost** [comment]
Memes most likely

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -209,12 +209,12 @@
 
 /obj/item/weapon/storage/box/flashes/New()
 	..()
-	new /obj/item/device/assembly/flash/handheld(src)
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/storage/boxes.dm:212]: Would be nice if this was a for loop. That means easier to see how many items and easier to change to a different amount

**Comment by JohnGinnane** [comment]
How did you change am the flashes in the map? I think it's updated then but I noticed the index for the tiles is different on some

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 2]
```diff
@@ -286,7 +286,7 @@
...
 "afB" = (/turf/simulated/wall,/area/security/lockers)
-"afC" = (/obj/machinery/vending/security{products = list(/obj/item/weapon/restraints/handcuffs = 15, /obj/item/weapon/restraints/handcuffs/cable/zipties = 10, /obj/item/weapon/grenade/flashbang = 4, /obj/item/device/assembly/flash/handheld = 5, /obj/item/weapon/reagent_containers/food/snacks/donut = 12, /obj/item/weapon/storage/box/evidence = 6, /obj/item/device/flashlight/seclite = 4)},/turf/simulated/floor/plasteel{icon_state = "showroomfloor"},/area/security/lockers)
+"afC" = (/obj/machinery/vending/security{products = list(/obj/item/weapon/restraints/handcuffs = 15, /obj/item/weapon/restraints/handcuffs/cable/zipties = 10, /obj/item/weapon/grenade/flashbang = 4, /obj/item/device/assembly/flash = 5, /obj/item/weapon/reagent_containers/food/snacks/donut = 12, /obj/item/weapon/storage/box/evidence = 6, /obj/item/device/flashlight/seclite = 4)},/turf/simulated/floor/plasteel{icon_state = "showroomfloor"},/area/security/lockers)
```
Comment (JohnGinnane) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm:289]: This looks good

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 3]
```diff
@@ -5177,7 +5178,6 @@
 "bXg" = (/obj/structure/disposalpipe/segment{dir = 2; icon_state = "pipe-c"},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 4},/turf/simulated/floor/plating,/area/maintenance/aft)
 "bXh" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/obj/structure/disposalpipe/segment{dir = 4; icon_state = "pipe-c"},/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 4},/turf/simulated/floor/plating,/area/maintenance/aft)
 "bXi" = (/obj/structure/disposalpipe/segment{dir = 8; icon_state = "pipe-c"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"},/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 10},/turf/simulated/floor/plating,/area/maintenance/aft)
-"bXj" = (/obj/structure/table,/obj/item/device/flashlight{pixel_x = 1; pixel_y = 5},/obj/item/device/flashlight{pixel_x = 1; pixel_y = 5},/obj/item/device/assembly/flash/handheld,/obj/item/device/assembly/flash/handheld,/obj/machinery/ai_status_display{pixel_x = -32; pixel_y = 0},/turf/simulated/floor/plating,/area/storage/tech)
```
Comment (JohnGinnane) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm:5180]: I'm worried bXj will be missing and is replaced with a different index

**Comment by ghost** [comment]
I just went through the files in notepad++ and did a search and replace.

**Comment by JohnGinnane** [comment]
Yeah never mind, I see how it's saved in the map. Everything looks good

**Comment by JohnGinnane** [comment]
Do we need head input on code refactoring?

**Comment by McBawbaggings** [comment]
Coffee refactoring?
I'm not sure what that's about.

**Comment by JohnGinnane** [comment]
Fixed.

**Comment by Nebyab** [comment]
ok

**Comment by Carbonhell** [comment]
No, head input is not needed for code refactoring. Merging.

**Pull request merged**